docs_sphinx/developer/guidelines/documentation.rst

```diff
@@ -4,7 +4,9 @@ Documentation
 It is very important to maintain documentation. We use the
 `Sphinx documentation generator <http://www.sphinx-doc.org/en/stable/>`__
 tools. The documentation is all hand written. Sphinx source files are stored in the
-``docs_sphinx`` folder.                                                      
+``docs_sphinx`` folder. The HTML files can be generated via the script
+``dev/tools/docs/build_html_brian2.py`` and end
+up in the ``docs`` folder (currently: ``dev/tools/docs``).
```

Comment:
We are almost there :) Can you remove the `(currently...)` part -- the documentation ends up in the `docs` folder directly in the main directory (i.e. parallel to ``docs_sphinx``), no need to add anything else.

Reply:
Oh!! that was really a myopic change... Sorry for that...